dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -78,35 +78,6 @@ def create_level_with_rubric
     )
   end
 
-  test "should return level_properties" do
```

Comment:
Not strictly related to this change but @breville @molly-moen I moved the test for `summarize_for_lab2_properties` into `level_test` so we didn't have to update two tests every time, and so we could test different branching paths within that function (like with `showRubric`)